Find_who_tweeted_the_most/Find_who_tweeted_the_most.py

```diff
@@ -144,11 +149,23 @@ def input_window():
         for index,name in enumerate(user_list):
             print(name+' '+tweet_id+str(index+1))
             
-        output_window(user_list) 
+        output_window(user_list)
+    print("\nOutput : ")
+    output_list.sort()

```

Comment:
storing all the outputs on a list & sorting out before dispatching to the monitor .